packages/react-components/react-menu/src/components/MenuItemLink/renderMenuItemLink.tsx

```diff
@@ -0,0 +1,23 @@
+/** @jsxRuntime classic */
+/** @jsx createElement */
+
+import { createElement } from '@fluentui/react-jsx-runtime';
+import { getSlotsNext } from '@fluentui/react-utilities';
+import type { MenuItemLinkState, MenuItemLinkSlots } from './MenuItemLink.types';
+
+/**
+ * Render the final JSX of MenuItemLink
+ */
+export const renderMenuItemLink_unstable = (state: MenuItemLinkState) => {
```

Comment:
Can't you do `renderMenuItemLink_unstable = renderMenuLink_unstable`?

Reply:
yeah, this is possible, I initially did it to avoid casting the state type, but it makes sense